.github/workflows/linting.yml

```diff
@@ -0,0 +1,25 @@
+name: Linting
```

Comment:
I am wondering if this can be a pre-commit step? Just asking if this is necessary step to do here